docs/en_US/Compression/QuantizationSpeedup.rst

```diff
@@ -0,0 +1,113 @@
+Speed up Mixed Precision Quantization Model (experimental)
+==========================================================
+
+
+Introduction
+------------
+
+Deep learning network has been computational intensive and memory intensive 
+which increase the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like haq(https://arxiv.org/pdf/1811.08886.pdf) only support simulated mixed precision quantization which will 
```

Comment:
the hyper-link is not supposed to be added like this...

Reply:
Modified.